gradle.properties

```diff
@@ -42,3 +42,4 @@ org.gradle.jvmargs=--add-exports jdk.compiler/com.sun.tools.javac.api=ALL-UNNAME
   --add-exports jdk.compiler/com.sun.tools.javac.parser=ALL-UNNAMED \
   --add-exports jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED \
   --add-exports jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED
+GITHUB_ACCESS_TOKEN=null
```

Comment:
From https://github.com/department-of-veterans-affairs/lighthouse-di-demo-java-starterkit/commit/d48fecf8a622c3a397f37ddaf9416237efc17f28#diff-3d103fc7c312a3e136f88e81cef592424b8af2464c468116545c4d22d6edcf19